src/pke/lib/schemebase/base-pke.cpp

```diff
@@ -201,8 +202,25 @@ std::shared_ptr<std::vector<Element>> PKEBase<Element>::EncryptZeroCore(
                   ? Element(dgg, elementParams, Format::EVALUATION)
                   : Element(tug, elementParams, Format::EVALUATION);
 
-  Element e0(dgg, elementParams, Format::EVALUATION);
-  Element e1(dgg, elementParams, Format::EVALUATION);
+  //Element e0(dgg, elementParams, Format::EVALUATION);
```

Comment:
remove these commented lines

Reply:
fixed

Reply:
don't mark it fixed unless it is in the branch